src/Sylius/Behat/Context/Transform/CurrencyContext.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Currency\Converter\CurrencyNameConverterInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+final class CurrencyContext implements Context
+{
+    /**
+     * @var CurrencyNameConverterInterface
+     */
+    private $converter;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $repository;
+
+    /**
+     * @param CurrencyNameConverterInterface $converter
+     * @param RepositoryInterface $repository
+     */
+    public function __construct(
+        CurrencyNameConverterInterface $converter,
+        RepositoryInterface $repository
+    ) {
+        $this->converter = $converter;
+        $this->repository = $repository;
+    }
+
+    /**
+     * @Transform /^currency "([^"]+)"$/
```

Comment:
If you would apply my [comment](https://github.com/Sylius/Sylius/pull/4604/files#r58204567) it can be written as @Transform :currency